src/GameServer/MessageHandler/TargetedSkillHandlerPlugIn.cs

```diff
@@ -13,21 +13,41 @@ namespace MUnique.OpenMU.GameServer.MessageHandler;
 using MUnique.OpenMU.PlugIns;
 
 /// <summary>
-/// Handler for targeted skill packets.
+/// Reprersents the targeted skill packet handler.
+/// </summary>
+internal partial class TargetedSkillHandlerPlugIn
```

Comment:
Why partial in the same file?

Reply:
I was trying to segregate the interface implementation, and own implementation.
I think it was because of the code compliance warnings about arrangement of fields that i decided to do partials

Reply:
Now removed.